### DISCUSSION TITLE
AMI packages needed for?

### DISCUSSION THREAD

**Discussion by stevenGravy** [gravitational/teleport#5649] [Q&A]
The AMI packages install includes this comment for the package.  Are those packages outside of `uuid` solely for grafana?

```bash
# Install uuid used for random token generation, nginx for grafana frontend
yum install -y uuid libffi-devel gcc openssl-devel adduser libfontconfig
```

**Comment by webvictim** [comment]
`adduser` is used for adding the Teleport user into the AMI.

I'm not 100% sure exactly why all the other packages are there, we can look into whether they can be removed. We install `certbot` and a few other things via `pip`, which has some requirements to be able to compile binary packages.
